### PR TITLE
Ck editor fix selectors

### PR DESCRIPTION
**Pull request by chessky**

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 7 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [VisualEditModePageObject.java#L153](https://github.com/Wikia/selenium-tests/blob/1f695c933b2f5a9a66dac8a32742ff5bba5eae7e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java#L153): Remove this unused private "verifyComponent" method. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
1. ![MAJOR][MAJOR] [VisualEditModePageObject.java#L280](https://github.com/Wikia/selenium-tests/blob/1f695c933b2f5a9a66dac8a32742ff5bba5eae7e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java#L280): Reduce this lambda expression number of lines from 22 to at most 20. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1188)
1. ![MAJOR][MAJOR] [VisualEditModePageObject.java#L319](https://github.com/Wikia/selenium-tests/blob/1f695c933b2f5a9a66dac8a32742ff5bba5eae7e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java#L319): Either log or rethrow this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)
1. ![MINOR][MINOR] [ForumBoardPage.java#L193](https://github.com/Wikia/selenium-tests/blob/1f695c933b2f5a9a66dac8a32742ff5bba5eae7e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumBoardPage.java#L193): Define a constant instead of duplicating this literal ") li.follow" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [ForumBoardPage.java#L193](https://github.com/Wikia/selenium-tests/blob/1f695c933b2f5a9a66dac8a32742ff5bba5eae7e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumBoardPage.java#L193): Define a constant instead of duplicating this literal ".thread:nth-child(" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [ForumBoardPage.java#L197](https://github.com/Wikia/selenium-tests/blob/1f695c933b2f5a9a66dac8a32742ff5bba5eae7e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumBoardPage.java#L197): Define a constant instead of duplicating this literal "unfollowIfDiscussionIsFollowed" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![INFO][INFO] [ForumBoardPage.java#L192](https://github.com/Wikia/selenium-tests/blob/1f695c933b2f5a9a66dac8a32742ff5bba5eae7e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumBoardPage.java#L192): Complete the task associated to this TODO comment. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 7 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [VisualEditModePageObject.java#L153](https://github.com/Wikia/selenium-tests/blob/171d16a82c21b593d3b2ccaeb5725cf190e750d9/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java#L153): Remove this unused private "verifyComponent" method. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
1. ![MAJOR][MAJOR] [VisualEditModePageObject.java#L280](https://github.com/Wikia/selenium-tests/blob/171d16a82c21b593d3b2ccaeb5725cf190e750d9/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java#L280): Reduce this lambda expression number of lines from 22 to at most 20. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1188)
1. ![MAJOR][MAJOR] [VisualEditModePageObject.java#L319](https://github.com/Wikia/selenium-tests/blob/171d16a82c21b593d3b2ccaeb5725cf190e750d9/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java#L319): Either log or rethrow this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)
1. ![MINOR][MINOR] [ForumBoardPage.java#L193](https://github.com/Wikia/selenium-tests/blob/171d16a82c21b593d3b2ccaeb5725cf190e750d9/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumBoardPage.java#L193): Define a constant instead of duplicating this literal ") li.follow" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [ForumBoardPage.java#L193](https://github.com/Wikia/selenium-tests/blob/171d16a82c21b593d3b2ccaeb5725cf190e750d9/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumBoardPage.java#L193): Define a constant instead of duplicating this literal ".thread:nth-child(" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [ForumBoardPage.java#L197](https://github.com/Wikia/selenium-tests/blob/171d16a82c21b593d3b2ccaeb5725cf190e750d9/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumBoardPage.java#L197): Define a constant instead of duplicating this literal "unfollowIfDiscussionIsFollowed" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![INFO][INFO] [ForumBoardPage.java#L192](https://github.com/Wikia/selenium-tests/blob/171d16a82c21b593d3b2ccaeb5725cf190e750d9/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumBoardPage.java#L192): Complete the task associated to this TODO comment. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 7 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [VisualEditModePageObject.java#L153](https://github.com/Wikia/selenium-tests/blob/8a3c03524428b5a42a5f2f17e63e82bc772ceb89/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java#L153): Remove this unused private "verifyComponent" method. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
1. ![MAJOR][MAJOR] [VisualEditModePageObject.java#L280](https://github.com/Wikia/selenium-tests/blob/8a3c03524428b5a42a5f2f17e63e82bc772ceb89/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java#L280): Reduce this lambda expression number of lines from 22 to at most 20. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1188)
1. ![MAJOR][MAJOR] [VisualEditModePageObject.java#L319](https://github.com/Wikia/selenium-tests/blob/8a3c03524428b5a42a5f2f17e63e82bc772ceb89/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java#L319): Either log or rethrow this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)
1. ![MINOR][MINOR] [ForumBoardPage.java#L193](https://github.com/Wikia/selenium-tests/blob/8a3c03524428b5a42a5f2f17e63e82bc772ceb89/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumBoardPage.java#L193): Define a constant instead of duplicating this literal ") li.follow" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [ForumBoardPage.java#L193](https://github.com/Wikia/selenium-tests/blob/8a3c03524428b5a42a5f2f17e63e82bc772ceb89/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumBoardPage.java#L193): Define a constant instead of duplicating this literal ".thread:nth-child(" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [ForumBoardPage.java#L197](https://github.com/Wikia/selenium-tests/blob/8a3c03524428b5a42a5f2f17e63e82bc772ceb89/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumBoardPage.java#L197): Define a constant instead of duplicating this literal "unfollowIfDiscussionIsFollowed" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![INFO][INFO] [ForumBoardPage.java#L192](https://github.com/Wikia/selenium-tests/blob/8a3c03524428b5a42a5f2f17e63e82bc772ceb89/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumBoardPage.java#L192): Complete the task associated to this TODO comment. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 7 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [VisualEditModePageObject.java#L153](https://github.com/Wikia/selenium-tests/blob/ae4aadf2223ed3ca72648775f88c37f834c7c71e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java#L153): Remove this unused private "verifyComponent" method. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
1. ![MAJOR][MAJOR] [VisualEditModePageObject.java#L280](https://github.com/Wikia/selenium-tests/blob/ae4aadf2223ed3ca72648775f88c37f834c7c71e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java#L280): Reduce this lambda expression number of lines from 22 to at most 20. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1188)
1. ![MAJOR][MAJOR] [VisualEditModePageObject.java#L319](https://github.com/Wikia/selenium-tests/blob/ae4aadf2223ed3ca72648775f88c37f834c7c71e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java#L319): Either log or rethrow this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)
1. ![MINOR][MINOR] [ForumBoardPage.java#L193](https://github.com/Wikia/selenium-tests/blob/ae4aadf2223ed3ca72648775f88c37f834c7c71e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumBoardPage.java#L193): Define a constant instead of duplicating this literal ") li.follow" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [ForumBoardPage.java#L193](https://github.com/Wikia/selenium-tests/blob/ae4aadf2223ed3ca72648775f88c37f834c7c71e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumBoardPage.java#L193): Define a constant instead of duplicating this literal ".thread:nth-child(" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [ForumBoardPage.java#L197](https://github.com/Wikia/selenium-tests/blob/ae4aadf2223ed3ca72648775f88c37f834c7c71e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumBoardPage.java#L197): Define a constant instead of duplicating this literal "unfollowIfDiscussionIsFollowed" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![INFO][INFO] [ForumBoardPage.java#L192](https://github.com/Wikia/selenium-tests/blob/ae4aadf2223ed3ca72648775f88c37f834c7c71e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumBoardPage.java#L192): Complete the task associated to this TODO comment. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/test/java/com/wikia/webdriver/testcases/mediatests/addvideo/VetArticlePlaceholderTests.java [hunk 1]
```diff
@@ -46,6 +46,7 @@ public void userCanAddVideoFromWikiaUsingPlaceholder() {
 
   @Test(groups = {"VideoArticlePlacehoder_003"})
   @Execute(asUser = User.SUS_STAFF, onWikia = "mobileregressiontesting")
+  @RelatedIssue(issueID = "XW-4584")
```
Comment (rybmat) [inline src/test/java/com/wikia/webdriver/testcases/mediatests/addvideo/VetArticlePlaceholderTests.java]: I think this should be fixed on prod already, could you check if test passes and remove this annotation?

File: src/test/java/com/wikia/webdriver/testcases/mediatests/addvideo/VetArticlePlaceholderTests.java [hunk 2]
```diff
@@ -66,6 +67,7 @@ public void userCanAddVideoByUrlUsingPlaceholderInEditMode() {
 
   @Test(groups = {"VideoArticlePlacehoder_004"})
   @Execute(asUser = User.SUS_STAFF, onWikia = "mobileregressiontesting")
+  @RelatedIssue(issueID = "XW-4584")
```
Comment (rybmat) [inline src/test/java/com/wikia/webdriver/testcases/mediatests/addvideo/VetArticlePlaceholderTests.java]: I think this should be fixed on prod already, could you check if test passes and remove this annotation?

File: src/test/java/com/wikia/webdriver/testcases/mediatests/modal/VetModalCaptionTests.java [hunk 3]
```diff
@@ -34,6 +35,7 @@
 
   @Test(groups = {"VetModalCaption_001"})
   @Execute(asUser = User.USER)
+  @RelatedIssue(issueID = "XW-4586", comment = "broke after update of CKE")
```
Comment (rybmat) [inline src/test/java/com/wikia/webdriver/testcases/mediatests/modal/VetModalCaptionTests.java]: https://github.com/Wikia/app/pull/14644
We have fix for this in QA, could you check if this test passes after this fix? https://github.com/Wikia/app/pull/14644

**Comment by WikiaTech** [comment]
SonarQube analysis reported 7 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [VisualEditModePageObject.java#L153](https://github.com/Wikia/selenium-tests/blob/9d857a1e39ebceff6cb623fc4cd8aa6161b71c6c/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java#L153): Remove this unused private "verifyComponent" method. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
1. ![MAJOR][MAJOR] [VisualEditModePageObject.java#L280](https://github.com/Wikia/selenium-tests/blob/9d857a1e39ebceff6cb623fc4cd8aa6161b71c6c/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java#L280): Reduce this lambda expression number of lines from 22 to at most 20. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1188)
1. ![MAJOR][MAJOR] [VisualEditModePageObject.java#L319](https://github.com/Wikia/selenium-tests/blob/9d857a1e39ebceff6cb623fc4cd8aa6161b71c6c/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java#L319): Either log or rethrow this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)
1. ![MINOR][MINOR] [ForumBoardPage.java#L193](https://github.com/Wikia/selenium-tests/blob/9d857a1e39ebceff6cb623fc4cd8aa6161b71c6c/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumBoardPage.java#L193): Define a constant instead of duplicating this literal ") li.follow" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [ForumBoardPage.java#L193](https://github.com/Wikia/selenium-tests/blob/9d857a1e39ebceff6cb623fc4cd8aa6161b71c6c/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumBoardPage.java#L193): Define a constant instead of duplicating this literal ".thread:nth-child(" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [ForumBoardPage.java#L197](https://github.com/Wikia/selenium-tests/blob/9d857a1e39ebceff6cb623fc4cd8aa6161b71c6c/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumBoardPage.java#L197): Define a constant instead of duplicating this literal "unfollowIfDiscussionIsFollowed" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![INFO][INFO] [ForumBoardPage.java#L192](https://github.com/Wikia/selenium-tests/blob/9d857a1e39ebceff6cb623fc4cd8aa6161b71c6c/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumBoardPage.java#L192): Complete the task associated to this TODO comment. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/test/java/com/wikia/webdriver/testcases/mediatests/modal/VetModalCaptionTests.java [hunk 4]
```diff
@@ -4,6 +4,7 @@
 import com.wikia.webdriver.common.contentpatterns.VideoContent;
 import com.wikia.webdriver.common.core.Assertion;
 import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.annotations.RelatedIssue;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/mediatests/modal/VetModalCaptionTests.java]: MINOR Remove this unused import 'com.wikia.webdriver.common.core.annotations.RelatedIssue'. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 8 issues
* ![MAJOR][MAJOR] 3 major
* ![MINOR][MINOR] 4 minor
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

#### 7 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [VisualEditModePageObject.java#L153](https://github.com/Wikia/selenium-tests/blob/3490ea5295b93d41bcf69e5e21a13193c3c5576e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java#L153): Remove this unused private "verifyComponent" method. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
1. ![MAJOR][MAJOR] [VisualEditModePageObject.java#L280](https://github.com/Wikia/selenium-tests/blob/3490ea5295b93d41bcf69e5e21a13193c3c5576e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java#L280): Reduce this lambda expression number of lines from 22 to at most 20. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1188)
1. ![MAJOR][MAJOR] [VisualEditModePageObject.java#L319](https://github.com/Wikia/selenium-tests/blob/3490ea5295b93d41bcf69e5e21a13193c3c5576e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java#L319): Either log or rethrow this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)
1. ![MINOR][MINOR] [ForumBoardPage.java#L193](https://github.com/Wikia/selenium-tests/blob/3490ea5295b93d41bcf69e5e21a13193c3c5576e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumBoardPage.java#L193): Define a constant instead of duplicating this literal ") li.follow" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [ForumBoardPage.java#L193](https://github.com/Wikia/selenium-tests/blob/3490ea5295b93d41bcf69e5e21a13193c3c5576e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumBoardPage.java#L193): Define a constant instead of duplicating this literal ".thread:nth-child(" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [ForumBoardPage.java#L197](https://github.com/Wikia/selenium-tests/blob/3490ea5295b93d41bcf69e5e21a13193c3c5576e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumBoardPage.java#L197): Define a constant instead of duplicating this literal "unfollowIfDiscussionIsFollowed" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![INFO][INFO] [ForumBoardPage.java#L192](https://github.com/Wikia/selenium-tests/blob/3490ea5295b93d41bcf69e5e21a13193c3c5576e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumBoardPage.java#L192): Complete the task associated to this TODO comment. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 8 issues
* ![MAJOR][MAJOR] 3 major
* ![MINOR][MINOR] 4 minor
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

#### 7 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [VisualEditModePageObject.java#L153](https://github.com/Wikia/selenium-tests/blob/faa8750d37015752eed111ddb130662be83d8d88/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java#L153): Remove this unused private "verifyComponent" method. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
1. ![MAJOR][MAJOR] [VisualEditModePageObject.java#L280](https://github.com/Wikia/selenium-tests/blob/faa8750d37015752eed111ddb130662be83d8d88/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java#L280): Reduce this lambda expression number of lines from 22 to at most 20. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1188)
1. ![MAJOR][MAJOR] [VisualEditModePageObject.java#L319](https://github.com/Wikia/selenium-tests/blob/faa8750d37015752eed111ddb130662be83d8d88/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java#L319): Either log or rethrow this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)
1. ![MINOR][MINOR] [ForumBoardPage.java#L193](https://github.com/Wikia/selenium-tests/blob/faa8750d37015752eed111ddb130662be83d8d88/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumBoardPage.java#L193): Define a constant instead of duplicating this literal ") li.follow" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [ForumBoardPage.java#L193](https://github.com/Wikia/selenium-tests/blob/faa8750d37015752eed111ddb130662be83d8d88/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumBoardPage.java#L193): Define a constant instead of duplicating this literal ".thread:nth-child(" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [ForumBoardPage.java#L197](https://github.com/Wikia/selenium-tests/blob/faa8750d37015752eed111ddb130662be83d8d88/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumBoardPage.java#L197): Define a constant instead of duplicating this literal "unfollowIfDiscussionIsFollowed" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![INFO][INFO] [ForumBoardPage.java#L192](https://github.com/Wikia/selenium-tests/blob/faa8750d37015752eed111ddb130662be83d8d88/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumBoardPage.java#L192): Complete the task associated to this TODO comment. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 7 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [VisualEditModePageObject.java#L153](https://github.com/Wikia/selenium-tests/blob/3b72dbb24bb351cd4efd3b649ca147f25f0083b3/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java#L153): Remove this unused private "verifyComponent" method. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
1. ![MAJOR][MAJOR] [VisualEditModePageObject.java#L280](https://github.com/Wikia/selenium-tests/blob/3b72dbb24bb351cd4efd3b649ca147f25f0083b3/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java#L280): Reduce this lambda expression number of lines from 22 to at most 20. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1188)
1. ![MAJOR][MAJOR] [VisualEditModePageObject.java#L319](https://github.com/Wikia/selenium-tests/blob/3b72dbb24bb351cd4efd3b649ca147f25f0083b3/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java#L319): Either log or rethrow this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)
1. ![MINOR][MINOR] [ForumBoardPage.java#L193](https://github.com/Wikia/selenium-tests/blob/3b72dbb24bb351cd4efd3b649ca147f25f0083b3/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumBoardPage.java#L193): Define a constant instead of duplicating this literal ") li.follow" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [ForumBoardPage.java#L193](https://github.com/Wikia/selenium-tests/blob/3b72dbb24bb351cd4efd3b649ca147f25f0083b3/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumBoardPage.java#L193): Define a constant instead of duplicating this literal ".thread:nth-child(" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [ForumBoardPage.java#L197](https://github.com/Wikia/selenium-tests/blob/3b72dbb24bb351cd4efd3b649ca147f25f0083b3/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumBoardPage.java#L197): Define a constant instead of duplicating this literal "unfollowIfDiscussionIsFollowed" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![INFO][INFO] [ForumBoardPage.java#L192](https://github.com/Wikia/selenium-tests/blob/3b72dbb24bb351cd4efd3b649ca147f25f0083b3/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumBoardPage.java#L192): Complete the task associated to this TODO comment. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**